src/charm.py

```diff
@@ -115,7 +115,7 @@ def _on_tctl_action(self, event):
             return
 
         # TODO(kelkawi-a): do not assume the app is always deployed with this name.
```

Comment:
I saw that you use `SERVER_APP_NAME` for this further below. Is this a good time to address this TODO as well?

Reply:
Good point, i've made the server name configurable.